module-services/service-cellular/Pdu.cpp

```diff
@@ -0,0 +1,982 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <cstdint>
+#include <variant>
+
+#include <Utils.hpp>
+
+#include "Pdu.hpp"
```

Comment:
According to our guidelines, the `Pdu.hpp` header should be first. https://github.com/mudita/MuditaOS/blob/master/doc/MuditaCppCodingGuidelines.md#m-3-sort-the-include-files

Reply:
Done